synth/src/cli/telemetry.rs

```diff
@@ -391,7 +395,11 @@ impl TelemetryClient {
 
         Self {
             ph_client: posthog_rs::client(API_KEY),
-            uuid: get_or_initialise_uuid(),
+            uuid: get_or_initialise_uuid().unwrap_or_else(|error| {
+                error
+                    .context("failed to get uuid for telemetry")
+                    .to_string()
+            }),
```

Comment:
I'm thinking of reworking this line, but would still like to send the error via telemetry somehow :thinking: